### PR TITLE
Add Caddy snippet for Pasthis

### PR DESCRIPTION
**Pull request by fr33tux**
Add [Pasthis](https://github.com/moulecorp/pasthis) configuration snippet.

### CODE REVIEW

File: pasthis/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	errors "/var/log/caddy/paste.yourdomain.net.errors.log" {
+		rotate_size 100
+	}
+}
+
+
+# Redirect to HTTPS
+http://paste.yourdomain.net {
+	redir https://paste.yourdomain.net{uri}
+}
```
Comment (mholt) [inline pasthis/Caddyfile]: Not necessary, Caddy does this redirect for you!

File: pasthis/Caddyfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Pasthis Caddy configuration
+# Requires no plugins
+# - https://caddyserver.com/download/linux/amd64
+# - https://caddyserver.com/download/linux/amd64/signature
+
+
+# TLS - defaults are sane
+tls admin@paste.yourdomain.net {
+        must_staple
+}
```
Comment (mholt) [inline pasthis/Caddyfile]: Just checking... did you try this Caddyfile? You can't have directives outside a site block.

Comment (fr33tux) [inline pasthis/Caddyfile]: Indeed, sorry I mis-paste the content of my import directives. Fixed!
It's currently running https://p.fr33tux.org/.

File: pasthis/Caddyfile (hunk 1 above)
Comment (fr33tux) [inline pasthis/Caddyfile]: :+1:

**Comment by fr33tux** [comment]
Requested changes applied.

**Comment by mholt** [comment]
Thanks @fr33tux! For our examples repo, we like to have the minimum required configuration so that people don't confuse what is necessary with what is somebody's preference.

Is this the minimum required configuration for Pasthis? (I'm looking at must_staple, log and errors directives.)

**Comment by fr33tux** [comment]
Understood and applied!

**Comment by mholt** [comment]
Beautiful - much better! Thanks.

**Comment by fr33tux** [comment]
Thanks!

**Pull request merged**